crates/processing_render/src/lib.rs

```diff
@@ -257,14 +271,17 @@ fn set_app(app: App) {
 
 /// Initialize the app, if not already initialized. Must be called from the main thread and cannot
 /// be called concurrently from multiple threads.
+/// asset_path is Optional because only python needs to use it.
 #[cfg(not(target_arch = "wasm32"))]
-pub fn init() -> error::Result<()> {
+pub fn init(config: Option<Config>) -> error::Result<()> {
```

Comment:
Prefer just defaulting something over option for things that can be default-able (like empty maps). None is maybe a *little* nicer but I think `init(Config::default())` is fine. Our rust examples are a little verbose anyway.